addons/l10n_be/i18n/en.po

```diff
@@ -0,0 +1,6279 @@
+#: model:account.account,name:l10n_be.1_a680
+#: model:account.account,name:l10n_be.2_a680
+#: model:account.account.template,name:l10n_be.a680
+msgid "Transfer to deferred taxes"
+msgstr ""
+
+#. module: l10n_be
+#: model:account.account,name:l10n_be.1_a689
+#: model:account.account,name:l10n_be.2_a689
+#: model:account.account.template,name:l10n_be.a689
+msgid "Transfer to untaxed reserves"
+msgstr ""
+
+#. module: l10n_be
+#: model:account.group,name:l10n_be.1_be_group_680
+#: model:account.group,name:l10n_be.2_be_group_680
+#: model:account.group.template,name:l10n_be.be_group_680
+msgid "Transferts aux impôts différés"
+msgstr ""
+
+#. module: l10n_be
+#: model:account.group,name:l10n_be.1_be_group_68
+#: model:account.group,name:l10n_be.2_be_group_68
+#: model:account.group.template,name:l10n_be.be_group_68
+msgid "Transferts aux impôts différés et aux réserves immunisées"
+msgstr ""
+
+#. module: l10n_be
+#: model:account.group,name:l10n_be.1_be_group_689
+#: model:account.group,name:l10n_be.2_be_group_689
+#: model:account.group.template,name:l10n_be.be_group_689
+msgid "Transferts aux réserves immunisées"
+msgstr ""
+
+#. module: l10n_be
+#: model:account.account,name:l10n_be.1_a101
+#: model:account.account,name:l10n_be.2_a101
+#: model:account.account.template,name:l10n_be.a101
+msgid "Uncalled capital"
+msgstr ""
+
+#. module: l10n_be
+#: model:account.account,name:l10n_be.1_a4210
+#: model:account.account,name:l10n_be.2_a4210
+#: model:account.account.template,name:l10n_be.a4210
+msgid ""
+"Unsubordinated debentures payable after more than one year falling due "
+"within one year - Convertible"
+msgstr ""
+
+#. module: l10n_be
+#: model:account.account,name:l10n_be.1_a4211
+#: model:account.account,name:l10n_be.2_a4211
+#: model:account.account.template,name:l10n_be.a4211
+msgid ""
+"Unsubordinated debentures payable after more than one year falling due "
+"within one year - Non convertible"
+msgstr ""
+
+#. module: l10n_be
+#: model:account.account,name:l10n_be.1_a1710
+#: model:account.account,name:l10n_be.2_a1710
+#: model:account.account.template,name:l10n_be.a1710
+msgid ""
+"Unsubordinated debentures with a remaining term of more than one year - "
+"Convertible bonds"
+msgstr ""
+
+#. module: l10n_be
+#: model:account.account,name:l10n_be.1_a1711
+#: model:account.account,name:l10n_be.2_a1711
+#: model:account.account.template,name:l10n_be.a1711
+msgid ""
+"Unsubordinated debentures with a remaining term of more than one year - Non "
+"convertible bonds"
+msgstr ""
+
+#. module: l10n_be
+#: model:account.account,name:l10n_be.1_a132
+#: model:account.account,name:l10n_be.2_a132
+#: model:account.account.template,name:l10n_be.a132
+msgid "Untaxed reserves"
+msgstr ""
+
+#. module: l10n_be
+#: model:account.report.line,name:l10n_be.tax_report_title_taxes_deductibles
+msgid "V Deductible"
+msgstr ""
+
+#. module: l10n_be
+#: model:account.report,name:l10n_be.tax_report_vat
+msgid "VAT Report"
+msgstr "VAT Return"
```

Comment:
@william-andre This term was translated

Reply:
From English to English? :thinking:

Reply:
Since it is in a stable version, I can't change the source term.

Reply:
You can, you just need to update the related po files: they are not on Transifex anyways.

Reply:
https://github.com/odoo/odoo/wiki/Translations#is-it-possible-to-modify-source-terms-in-odoo
Are you sure? They are on Transifex though ... https://www.transifex.com/odoo/odoo-16-l10n/l10n_be/

Reply:
Doesn't matter, just update the .pot file, and the relevant .po files.
The reason the guideline exists is because you wouldn't be able to translate in all the supported languages of Odoo standard by yourself, which is not the case for this l10n.


This guideline doesn't apply for l10n modules

Reply:
@william-andre Okay, changed the source term.